scripts/release/publish-docs-artifacts.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Publish material2 docs assets to the material2-docs-content repo
+# material.angular.io will pull from this assets repo to get the latest docs
+
+cd "$(dirname $0)/../../"
+
+docsPath="./dist/docs"
+tempRepoPath="./dist/tempRepo"
+repoUrl="https://github.com/angular/material2-docs-content"
```

Comment:
It would be good to let this do a few test runs before committing to the real repo. We did this for the material2-builds; you can do the same by:
* Fork angular/material2-docs-content
* Add an access token for said fork to our Travis CI environment
* Update this script to use that token instead of `MATERIAL2_BUILDS_TOKEN`

We can then just let it run for a few commits and make sure it looks good before updating the repo / access token.